pages/spec/manifest.md

```diff
@@ -530,6 +573,28 @@ For more verbose layout use normal tables rather than inline tables to specify d
 git = "https://github.com/toml-f/toml-f"
 rev = "2f5eaba864ff630ba0c3791126a3f811b6e437f3"
 ```
+### Dependencies from a registry
+
+#### Namespace
+
+Packages obtained from a registry (both remote and local) are required to specify a namespace, which provides a way to uniquely identify and differentiate packages with identical names. The namespace is declared in the manifest (`fpm.toml`).
+
+```toml
+[dependencies]
+my-package.namespace = "my-namespace"
+```
+
+This will prompt fpm to download the newest version of "my-package", which belongs to "my-namespace", from the registry.
+
+#### Version
+
+If you want to download a specific version of a package instead of the newest one available, you can specify the version (`v`) in the manifest.
+
+```toml
+[dependencies]
+example-package.namespace = "example-namespace"
+example-package.v = "1.0.0"
+```
```

Comment:
This is kind of an interesting design. Not exactly what I would have initially thought of, but I suppose it works.